docs/tutorial/Split_Learning_for_bank_marketing.ipynb

```diff
@@ -1520,6 +1106,133 @@
     "             dp_spent_step_freq=dp_spent_step_freq,)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Let's visualize the training process"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
```

Comment:
The metrics here must be consistent with the results in the `summary`. You need to modify the results below in the summary as well

Reply:
Epochs in standalone version be consistent with those in the federated version